canonical/vanilla-framework
# Add form legend required symbol

## Done

- Add `is-required` class to fieldset legend

Fixes [list issues/bugs if needed]

## QA

- Open https://vanilla-framework-5214.demos.haus/docs/examples/patterns/forms/fieldset-required?theme=light
- See that the asterisk shows for fieldset legend

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

[if relevant, include a screenshot or screen capture]
